conans/test/integration/toolchains/env/test_virtualenv_powershell.py

```diff
@@ -114,3 +114,53 @@ def test(self):
     assert "MYPS1!!!!" in client.out
     assert "MYVC_CUSTOMVAR1=PATATA1" in client.out
     assert "MYVC_CUSTOMVAR2=PATATA2" in client.out
+
+@pytest.mark.skipif(platform.system() != "Windows", reason="Requires Windows powershell")
+def test_vcvars():
```

Comment:
This test is NOT running in CI because it is Window only and "integration" tests in Windows do not pass for regular PR, only when merged.

Reply:
Good point, probably those deserve to be moved to "functional", try to do it, hopefully github diff will be good and not show a huge remove/add diff